app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_many :created_events, class_name: 'Event', foreign_key: 'creator_id'
+  has_many :attendances, foreign_key: 'attendee_id'
```

Comment:
We don't have any kind of validation on creating new user, I was able to create a new user with invalid email format. Let's add necessary validation for email presence and format.

Reply:
fixed bay add name, email, password_digest validations